xbmc/Application.h

```diff
@@ -502,6 +501,7 @@ class CApplication : public CXBApplicationEx, public IPlayerCallback, public IMs
   std::atomic_uint m_WaitingExternalCalls;        /*!< counts threads wich are waiting to be processed in FrameMove */
   unsigned int m_ProcessedExternalCalls;          /*!< counts calls wich are processed during one "door open" cycle in FrameMove */
   unsigned int m_ProcessedExternalDecay = 0;      /*!< counts to close door after a few frames of no python activity */
+  std::shared_ptr<CApplicationPlayer> m_pPlayer;
```

Comment:
now thinking more about it I'd say that a pointer makes no sense here. It never gets deleted and nothing checks the pointer for being valid. Better make it a normal member. CAppPlayer wraps the actual player and does the checks for nullptr.

Reply:
Ok will change it to be a normal member like we did for CSeekHandler

Reply:
done + force-pushed